hashtable.csv

```diff
@@ -2,6 +2,6 @@ code, name
 0x11, sha1
 0x12, sha2-256
 0x13, sha2-512
-0x14, sha3
+0x14, sha3-256; KECCAK[512]
```

Comment:
i think this should be aliased to `sha3-512`, see https://github.com/jbenet/multihash/pull/22